storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.10.rst

```diff
@@ -0,0 +1,37 @@
+===========================
+|Percona XtraBackup| 2.4.10
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.10 on
+March 28, 2018. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.10/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.4
+series.
+
+Starting from now, |Percona XtraBackup| issue tracking system was moved from
+launchpad to `JIRA <https://jira.percona.com/projects/PXB>`_.
+All Percona software is open-source and free.
+
+Bugs fixed
+==========
+
+* Decrypt operation was failing due to regression in |Percona XtraBackup| 2.4.9.
```

Comment:
xbcrypt with "--encrypt-key-file seems" option was failing. Please check bug reference, is it really PXC- issue?